src/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -315,15 +315,7 @@ public override string ToString()
                 if (Length < int.MaxValue)
                 {
 #if !FEATURE_PORTABLE_SPAN
-                    return string.Create((int)Length, charSequence, (span, sequence) =>
-                    {
-                        foreach (ReadOnlyMemory<char> readOnlyMemory in sequence)
-                        {
-                            ReadOnlySpan<char> sourceSpan = readOnlyMemory.Span;
-                            sourceSpan.CopyTo(span);
-                            span = span.Slice(sourceSpan.Length);
-                        }
-                    });
+                    return string.Create((int)Length, charSequence, (span, sequence) => sequence.CopyTo(span));
```

Comment:
Please open a new PR with just this change since the fix to Seek is already in. Thanks.